content/en/users/online-storage/object-storage/_index.md

```diff
@@ -2,127 +2,231 @@
+(CLI) can be used to perform operations on the SWIFT endpoints available in
+the EGI Federated Cloud. This means using the command `fedcloud openstack`
+to query and manipulate storage containers and objects.
+
+{{% alert title="Note" color="info" %}} See
+[here](https://docs.openstack.org/python-openstackclient/latest/cli/command-objects/container.html)
+for documentation on all storage container-related commands, and
+[here](https://docs.openstack.org/python-openstackclient/latest/cli/command-objects/object.html)
+for all object-related commands.
+{{% /alert %}}
+
+#### List storage containers
+
+For example, to access to the SWIFT endpoint at IFCA-LCG2 via the
+Pilot VO (vo.access.egi.eu), use the following  FedCloud command
+to list the available storage containers:
+
+{{% alert title="Tip" color="info" %}} Instead of passing the site, VO, etc.
+on the command line each time, you can use
+[FedCloud CLI environment variables](https://fedcloudclient.fedcloud.eu/usage.html#environment-variables).
+{{% /alert %}}
 
 ```shell
-$ fedcloud openstack --site IFCA-LCG2 --vo vo.access.egi.eu container list
+$ export EGI_SITE=IFCA-LCG2
+$ export EGI_VO=vo.access.egi.eu
+$ fedcloud openstack container list
 +------------------+
 | Name             |
 +------------------+
 | egi_endorsed_vas |
 +------------------+
 ```
 
-Creating a new container:
+#### Create new storage container
 
 ```shell
-$ fedcloud openstack --site IFCA-LCG2 --vo vo.access.egi.eu container create test-egi
+$ fedcloud openstack container create test-egi
 +---------+-----------+------------------------------------------------------+
 | account | container | x-trans-id                                           |
 +---------+-----------+------------------------------------------------------+
 | v1      | test-egi  | tx000000000000000000afc-005f845160-2bb3ed4-RegionOne |
 +---------+-----------+------------------------------------------------------+
 ```
 
-Creating a new Object by uploading a file:
+#### Create new object by uploading a file
 
 ```shell
-$ fedcloud openstack --site IFCA-LCG2 --vo vo.access.egi.eu \
-           object create test-egi file1.txt
+$ fedcloud openstack object create test-egi file1.txt
 +-----------+-----------+----------------------------------+
 | object    | container | etag                             |
 +-----------+-----------+----------------------------------+
 | file1.txt | test-egi  | 5bbf5a52328e7439ae6e719dfe712200 |
 +-----------+-----------+----------------------------------+
 ```
 
-Listing objects inside a container:
+#### List objects in a storage container
 
 ```shell
-$ fedcloud openstack --site IFCA-LCG2 --vo vo.access.egi.eu object list test-egi
+$ fedcloud openstack object list test-egi
 +-----------+
 | Name      |
 +-----------+
 | file1.txt |
 +-----------+
 ```
 
-Download an object:
+#### Download (the content of) an object
+
+<!-- markdownlint-disable commands-show-output -->
```

Comment:
Sorry for being a PITA, but it's the first time we encounter the need to disable `commands-show-output`, I'm also a bit puzzled with this, I feel adding the `$` makes things more noisy and less convenient for copy and pasting, so I would suggest to remove them as most of the time shell excerpt do not include them.
We can also document this in the style guide so that we have a clear rule for this.

Reply:
I understand. but I still think consistency is **much more important** for the reader. seeing the $ (or whatever we chose as the prompt) in that code block makes it clear that it is a command to run in a terminal. there are other such blocks in the docs that are not shell commands (and I intend to corrected them all). this so-called "rule" in my opinion is an arbitrary opinion of the linter authors, and is wrong/stupid (there are others as well).

if you want an example backing my opinion, try reading the docs of the commercial cloud providers. an example is the AWS docs, e.g. here https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-lamp-amazon-linux-2.html

however, if I get outvoted on this, I will do as you say.

Reply:
I agree that we need to have a coherent practice for this, until now everyone was on his own, that's why I said we need a clear rule.
The nice thing with the AWS doc is that the copy button is not copying the prompt.
I'm fine with enforcing having a prompt indicator if others also feel it's working better for the readers.
In order to move forward with this PR I would suggest you keep them and later we confirm this with others, update the style guide and plan to review all the pages for this.

Reply:
Fine, so let's close this. Let's try to trigger again this discussion when everyone will be back from the break.

Reply:
Let me update the pages with one more commit, then we can close it